Sources/Momento/CacheClient.swift

```diff
@@ -1,11 +1,255 @@
 import Foundation
 
-public enum ScalarType {
+enum ScalarType {
     case string(String)
     case data(Data)
 }
 
-typealias CacheClientProtocol = ControlClientProtocol & DataClientProtocol
+// The CacheClient class provides user-friendly, public-facing methods
+// with default values for non-required parameters and overloaded functions
+// that accept String and Data values directly rather than ScalarTypes
+
+protocol CacheClientProtocol: ControlClientProtocol & DataClientProtocol {}
+
+extension CacheClientProtocol {
```

Comment:
protocol extension was re-added because default parameter values didn't work correctly in tests that expected CacheClientProtocol object instead of just CacheClient